### ISSUE TITLE
Add `--publish` & `--watch` options to `cml check create`

### ISSUE THREAD

**Issue by 0x2b3bfa0** [iterative/cml#1143]

**Comment by 0x2b3bfa0** [comment]
Not sure about the `cml check create --watch` option. Are checks mutable by definition? 🤔

**Comment by dacbd** [comment]
I agree that I don't think a `--watch` option makes sense. You might update a check's status throughout your workflow if the ci API's allow, but the "content" is only set at the end when you know the definitive state of said check /right?

**Comment by 0x2b3bfa0** [comment]
Yes, that was my impression.
